integration/test-mb/test-amqp/src/test/java/org/wso2/ei/mb/test/client/TopicSender.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License, 
+ * Version 2.0 (the "License"); you may not use this file except 
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package test.java.org.wso2.ei.mb.test.client;
+
+import org.testng.log4testng.Logger;
+import org.wso2.ei.mb.test.utils.ConfigurationConstants;
+import org.wso2.ei.mb.test.utils.ConfigurationReader;
+import org.wso2.ei.mb.test.utils.JMSAcknowledgeMode;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
+import javax.jms.JMSException;
+import javax.jms.TextMessage;
+import javax.jms.Topic;
+import javax.jms.TopicConnection;
+import javax.jms.TopicConnectionFactory;
+import javax.jms.TopicPublisher;
+import javax.jms.TopicSession;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+
+/**
+ * Message sender to the topics
+ */
+public class TopicSender {
+    private static Logger log = Logger.getLogger(TopicSender.class);
+
+    private static final String initialConnectionFactory = "org.wso2.andes.jndi.PropertiesFileInitialContextFactory";
+    private static final String connectionFactoryNamePrefix = "connectionfactory.";
+    private static final String queueNamePrefix = "topic.";
+    private static final String connectionFactoryName = "andesConnectionfactory";
```

Comment:
Can we move these to a helper class? These values can be reused in several places. `getTCPConnectionURL()` can also be moved.